test/release/examples/benchmarks/shootout/meteor-fast.chpl

```diff
@@ -269,13 +269,14 @@ proc searchParallel(in board=0, in pos=0, used=0, placed=0, firstPiece=0) {
       if allMasks[currentMask] {
         const mask = allMasks[currentMask];
 
-        const currentSolution: [piecesDom] int =
-          [firstPiece, mask, 0, 0, 0, 0, 0, 0, 0, 0];
 
-        begin with (in board, in pos)
+        begin with (in board, in pos) {
+          const currentSolution: [piecesDom] int =
+            [firstPiece, mask, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
I thought that our solution to handling that was to enclose the declaration of the variable being used and the begin statement in a sync statement to guarantee it did not get referenced outside of its scope.

Moving the variable to be local within the scope of the begin is fine with me, too.  Did you notice a performance impact for this change?